python/lsst/ip/isr/isrTask.py

```diff
@@ -2271,6 +2274,7 @@ def overscanCorrection(self, ccdExposure, amp):
         ampName = amp.getName()
 
         keyBase = "LSST ISR OVERSCAN"
+        metadata[f"{keyBase} UNITS {amp.getName()}"] = "adu"
```

Comment:
I don't think this is quite right.  The key needs to specify SERIAL or PARALLEL as well (because the new task does serial overscan in adu and parallel in electron (unless bootstrap mode).

Reply:
This is in the old task. Also is "LSST ISR OVERSCAN UNITS" supposed to refer to the units of the reported mean, median, std of the overscan--which I think is always in adu--or the units that the overscan correction was performed? This is not clear to me.

Reply:
Is this key used anywhere?  My point is that the names should be consistent if we're adding something new, and we should specify serial and parallel.
My assumption was that these UNITS refer to the units of the values in the header.

Reply:
But we should make that explicit somewhere!

Reply:
I'm not sure on the word order (maybe `UNITS` at the end?) and it shouldn't need the amp name and now I see that this is wrong: https://github.com/lsst/ip_isr/blob/c1d8f0290d0bbd0d32dcb30a8f9e67faa9475539/python/lsst/ip/isr/isrTaskLSST.py#L893